File: fullstackdotnet.domain/Evento.cs
using System;
using System.Collections.Generic;

namespace fullstackdotnet.domain
{
    public class Evento
    {
        public int Id { get; set; }
        public string Local { get; set; }
        public DateTime DataEvento { get; set; }
        public string Tema { get; set; }
        public int QtdPublico { get; set; }
        public string ImageUrl { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public string Conteudo { get; set; }
        public List<Lote> Lotes { get; set; }
        public List<RedeSocial> RedesSociais { get; set; }
        public List<PalestranteEvento> PalestrantesEventos { get; set; }
    }
}

File: fullstackdotnet-app/src/app/models/Evento.ts
export interface Evento {
  id?: number;
  local: string;
  dataEvento: string;
  tema: string;
  qtdPublico: number;
  imageUrl: string;
  telefone: string;
  email: string;
  conteudo: string;
  lotes: any;
  redesSociais: any;
  palestrantesEventos: any;
  includePalestrantes: boolean;
}
